fix: get api host from env variables (#6672)
vuestorefront/vue-storefront: packages/core/docs/getting-started/configuration.md

- [`.lintstagedrc`](https://github.com/okonet/lint-staged#configuration) configures lint runner that enforces code style before every Git commit. By default, it runs ESLint described above.

## Environment variables

> Every integration might have its own environmental variables described in documentation for specific integration.

Reviewer: ```suggestion
Every integration might have its own environmental variables described in the documentation.
```

Those described here are common for every integration.

> - `API_HOST` - a host and port of the server that internal API calls are being made, e.g. `example.com` or `localhost:3000`. It is recommended to not leave it empty for security reasons.

Reviewer: ```suggestion
- `API_HOST` - a host and port of the server that internal API calls are being made, e.g. `example.com` or `localhost:3000`. It is recommended **to not leave it empty** for security reasons.
```
